src/status_im/contexts/shell/share/wallet/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.contexts.shell.share.wallet.style
+  (:require
+    [legacy.status-im.ui.components.colors :as colors]))
+
+(defn indicator-wrapper-style
+  [is-active]
```

Comment:
By convention, we use question mark for boolean variables, so this prop should be `active?` instead of `is-active`

Reply:
That's good to know. Thanks @briansztamfater